telemetry/packages/ui/app/context/pods.tsx

```diff
@@ -152,7 +152,7 @@ export const PodsProvider = ({ children }: { children: React.ReactNode }) => {
           publish(
             'latency/request',
             JSON.stringify({
-              latency: new Date().getTime().toString(),
+              timestamp: new Date().getTime().toString(),
```

Comment:
here and elsewhere, are you remembering that all mqtt messages are of the format:
```json
{
  header : {
    timestamp: int,
    priority: int
  },
  payload: { rest of data }
}
```

Reply:
Ahhh I see. Will need to change this throughout the telemetry code